modules/internal/ChapelDomain.chpl

```diff
@@ -2282,6 +2282,7 @@ module ChapelDomain {
     }
 
     @chpldoc.nodoc
+    @unstable("expand() is unstable and its behavior may change in the future")
     proc expand(off: integral ...rank) do return expand(off);
```

Comment:
Does the user get an unstable warning when calling the above overload if we remove its @unstable attribute?

I would prefer to remove the attribute if possible, however feel free to leave it as-is.

Ditto the other functions.

Reply:
The behavior I see is that the unstable warning is generated only when the user directly calls a function, and not when a function calls another one of its own overloads. Therefore I added the attribute to all overloads individually.